site/content/configuration.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
       clientId: `0oa1f3ut0aKpdwap5357`,
       issuer: 'https://okta.nd.edu/oauth2/ausxosq06SDdaFNMB356',
     },
+    userContentPath: `https://b9mic83lu2.execute-api.us-east-1.amazonaws.com/prod/`,
```

Comment:
This seems like something that ought to be a part of an automated process which retrieves an environment-appropriate value. But maybe it already does that and you're just populating a default here, so I'll say it's fine for now.

Reply:
Yes, this along with several other settings are something that will at some point get moved into an environment variable when deployment for the website is sorted out a bit better.